google/cloud/pubsub/config-mocks.cmake.in

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CMakeFindDependencyMacro)
+find_dependency(google_cloud_cpp_pubsub)
```

Comment:
if this was something like:

```CMake
find_dependency(@GOOGLE_CLOUD_CPP_WE_NEED_A_NAME@)
```

then this file could be a generic file shared by all the libraries.

Maybe for a future PR?

Reply:
I have taken a mental note.

We would probably use a macro for just the service name (e.g. pubsub) because we include `@LIBRARY@-targets.cmake` below.